elements/pl-integer-input/pl-integer-input.mustache

```diff
@@ -47,7 +47,8 @@
 </script>
 <span id="pl-integer-input-submission-{{uuid}}">
     {{#label}}<span>{{label}}</span>{{/label}}
-    <a role="button" class="btn btn-light btn-sm" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}{{#raw_submitted_answer}}<hr><h5><small class='text-uppercase'>What you submitted</small></h5><pre>{{raw_submitted_answer}}</pre>{{/raw_submitted_answer}}">INVALID <span><i class="fa fa-question-circle" aria-hidden="true"></i></span></a>
+    {{#raw_submitted_answer}}<code class="user-output-invalid">{{raw_submitted_answer}}</code>{{/raw_submitted_answer}}
+    <a role="button" class="btn btn-light btn-sm" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}">INVALID <span><i class="fa fa-question-circle" aria-hidden="true"></i></span></a>
```

Comment:
I'm wondering whether there is a better label for this button that would make it clearer that it can be clicked.

Maybe: "(!) Invalid: show details" or something to indicate what clicking it will do (show details). Or maybe we should have a separate badge and button? That is, an "Invalid" badge, and next to it a "show details" or "help" or "why?" button?

Here I was thinking that "(!)" is some kind of error icon. Maybe https://fontawesome.com/icons/exclamation-triangle?style=solid or something like that?

Reply:
I wonder if even adding something as simple as a small dropdown icon to the button:

![image](https://user-images.githubusercontent.com/1790491/74095391-5db68480-4ab5-11ea-8ecc-1a172a6235f3.png)

would help signal to users that the element has more information?  I will try playing around with this and the exclamation mark and see what looks good.

Reply:
Sounds good, anything to better indicate that (1) the input is invalid, and (2) here is a button you can click for more info. I have no opinion on whether we should have one element or two or something else entirely.